### PR TITLE
10 case study pages made

### PR DESCRIPTION
**Pull request by Arfah**
Please merge these but don't delete my branch as i'm still working on it! THANKSSSSS

### CODE REVIEW

File: data/graduates.yml [hunk 1]
```diff
@@ -40,7 +35,29 @@
   summary: Full-time Mum to Junior Developer at Mergermarket
   testimonial: Makers Academy is an investment in yourself and to be honest, it’s quite cheap in the long run. It’s about you as a person and you are gaining skills that will be extremely useful for many, many years and when you look at it that way, it's actually a steal.
   cohort: Graduated October 2014
-  case_study_link: http://blog.makersacademy.com/ruth-earle-i-was-a-full-time-mum-for-7-years-and-before-that-i-had-my-own-franchise-business-to-junior-developer-at-merger-market/
+  case_study:
+      - question: What were you doing before joining Makers Academy?
+        answer: I was a full time mum for 7 years and before that I had my own franchise business.
```
Comment (pitchinvasion) [inline data/graduates.yml]: mum -> Mum

File: data/graduates.yml [hunk 2]
```diff
@@ -40,7 +35,29 @@
...
+      - question: How did you come across MA and what made you decide to apply?
+        answer: I came across MA through a bootcamp review website. I was deciding between Makers and GA. GA felt too international and a numbers game. Makers on the other hand felt non salesy and I liked that.
+      - question: What did you find difficult during the course?
+        answer: I had a lot of guilt as my mind was on the course so much so I didn't get to spend quality time with the children and any spare time I had I spent it coding. I also had to keep pushing through the times where I felt so stupid.
+      - question: What happened during / after graduation and where are you job wise?
+        answer: I carried on working on my code. Merger Market was my 7th interview. It was difficult as I wanted to always do more and being a mum I chose to spend more time with the kids and that slowed my progress. I also needed to build my confidence. So it took a while. I had a lot of disappointing interviews, i'd give myself a few days and go back to coding and I kept attending Makers and got lots of support which is very special.
```
Comment (pitchinvasion) [inline data/graduates.yml]: capitalise mum and i'd

File: data/graduates.yml [hunk 3]
```diff
@@ -13,26 +12,22 @@
...
     - question: What happened during / after graduation and where are you job wise?
       answer: I was an alumni helper for a few months. I worked on a couple of paid freelance projects while job hunting and kept learning. I now have a job at Deloitte Digital.
-    - question: Where are you now working?
-      answer: Deloitte Digital
     - question: How has your life changed since graduation?
       answer: My life has changed completely because in real terms my career has completely changed and I’ve got a great group of friends that came out of Makers. Doing the course hits your confidence but weirdly makes you much more confident.
     - question: What advice would you give to people thinking of changing career?
-      answer: Make sure you do your research and you’ve given yourself and make sure you budget so you can take time to pick your dream job but at the same time don’t over think it and overthink the choice.
+      answer: Make sure you do your research and make sure you budget so that you can take time to pick your dream job, but at the same time don’t overthink it and overthink the choice.
```
Comment (roidriscoll) [inline data/graduates.yml]: 'overthink' is used twice here.

File: data/graduates.yml [hunk 4]
```diff
@@ -56,15 +95,59 @@
...
+            answer: Working in start-ups was one thing. I've always enjoyed creating tangible things and getting involved with creative pursuits. Combined with my interest in technology, it was only natural that I would start getting my hands dirty with some code.
+          - question: What did you find difficult during the course?
+            answer: The workload for sure. It isn't called intensive for no reason. I've never worked so hard at something in my life. But do not be put off, you will learn to deal with it and you will become better for it.
+          - question: What happened during / after graduation and where are you job wise?
+            answer: Afterwards, I got involved with some freelance work and then landed a job at Deloitte Digital.
+          - question: How has your life changed since graduation?
+            answer: For the better. I feel happier. I have a skill which is as close to magic as it gets.
+          - question: What advice would you give to people thinking of changing career?
+            answer: There are 2 options. You either stay in a career that doesn't fulfil you, or you take action until you reach your desired goal.
```
Comment (pitchinvasion) [inline data/graduates.yml]: 2 -> two

File: data/graduates.yml [hunk 5]
```diff
@@ -40,7 +35,29 @@
   summary: Full-time Mum to Junior Developer at Mergermarket
   testimonial: Makers Academy is an investment in yourself and to be honest, it’s quite cheap in the long run. It’s about you as a person and you are gaining skills that will be extremely useful for many, many years and when you look at it that way, it's actually a steal.
   cohort: Graduated October 2014
-  case_study_link: http://blog.makersacademy.com/ruth-earle-i-was-a-full-time-mum-for-7-years-and-before-that-i-had-my-own-franchise-business-to-junior-developer-at-merger-market/
+  case_study:
+      - question: What were you doing before joining Makers Academy?
+        answer: I was a full time Mum for 7 years and before that I had my own franchise business.
```
Comment (roidriscoll) [inline data/graduates.yml]: 'full time' should be hyphenated, no?

File: data/graduates.yml [hunk 6]
```diff
@@ -105,7 +267,29 @@
...
+            answer: It is a huge investment but you'll get a skill that does not have value. Learning how to learn. Beside this, being a web Developer is one of the most paid jobs around the world and it is just going to get better.
+          - question: What was your biggest worry before joining Makers Academy and how do you feel about it now?
+            answer: I was worried that I couldn't get a job without a degree. Now I see people with degrees not getting the position I've applied for.
+          - question: Sum up your Makers Academy experience
+            answer: It was one of the best experiences of my life, pair programming with other people without any coding background made me feel more comfortable, and I learnt to break problems into smaller pieces in order to solve them better. The atmosphere was just magic, everyone is so positive and I was shocked after seeing how much I've actually done after 12 weeks (about 10 projects!). I have increased my communication skills and did some presentations in front of people. Managerial decisions on team projects and learnt a lot of useful tools for self reflection. I love Makers and I will always!
```
Comment (pitchinvasion) [inline data/graduates.yml]: "I have increased my communication skills and did some presentations in front of people. Managerial decisions on team projects and learnt a lot of useful tools for self reflection." this sentence doesn't read

**Comment by pitchinvasion** [comment]
@Arfah there's more content tweaks needed before we can merge....

**Comment by Arfah** [comment]
This is still my pull request... I don't think Jack pushed his? Will go
over it with him.

**Comment by Arfah** [comment]
I've made a few more changes using your comments. Please merge now!

**Comment by pitchinvasion** [comment]
:shipit:

**Pull request merged**